### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (89)

This is the slider component of my CL . Please review it!

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by amartya-rs — APPROVED** [review]
Good job on the slider. You can merge.

File: Components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+}
+
+.slider-section {
+    display: flex;
+    background-color: var(--bg-color);
+    gap: 1rem;
+    padding: 1rem;
+    border-radius: 0.25rem;
```
Comment (bhushannaikaroskar) [inline Components/Slider/slider.css]: maybe if value is too small you can use pixels as unit. If rem is requirement you can keep it as it is no issues.

**Review by bhushannaikaroskar — APPROVED** [review]
Hey Megha, code is clean, short and readable. Minor suggestion from my side do check before merging. Approving the merge

File: Components/Slider/slider.css (hunk 1 above)
Comment (Megha-Joshi) [reply Components/Slider/slider.css]: Ok. Thanks for the suggestion!

**Pull request merged**